index.html

```diff
@@ -54,7 +54,194 @@ <h1 class="title">
                 </li>
             </ul>
         </div>
-
+    </section>
+    <section id="projects">
+        <div class="card">
+            <img src="images/work1.svg" alt="Project 1" class="card-image">
+            <div class="card-body">
+                <h1 class="card-title">Tonic</h1>
+                <ul class="position">
+                    <li class="name">CANOPY</li>
+                    <li><img src="images/icons/bullet.svg" alt=""></li>
+                    <li class="status">Back End Dev</li>
+                    <li><img src="images/icons/bullet.svg" alt=""></li>
+                    <li class="date">2015</li>
+                </ul>
+                <p class="work-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li class="language"><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">html</a></li>
+                    <li class="language"><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">css</a></li>
+                    <li class="language"><a
+                            href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">javascript</a></li>
+                </ul>
+                <button class="card-button">See Project</button>
+            </div>
+        </div>
+        <div class="card">
+            <img src="images/work2.svg" alt="Project 1" class="card-image">
+            <div class="card-body">
+                <h1 class="card-title">Multi-Post <br> Stories</h1>
+                <ul class="position">
+                    <li class="name">CANOPY</li>
+                    <li><img src="images/icons/bullet.svg" alt=""></li>
+                    <li class="status">Back End Dev</li>
+                    <li><img src="images/icons/bullet.svg" alt=""></li>
+                    <li class="date">2015</li>
+                </ul>
+                <p class="work-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li class="language"><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">html</a></li>
+                    <li class="language"><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">css</a></li>
+                    <li class="language"><a
+                            href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">javascript</a></li>
+                </ul>
+                <button class="card-button">See Project</button>
+            </div>
+        </div>
+        <div class="card">
+            <img src="images/work3.svg" alt="Project 1" class="card-image">
+            <div class="card-body">
+                <h1 class="card-title">Tonic</h1>
```

Comment:
[ Optional ] Kindly I recommend you to use a header tag hx(h2) for the project title (e.g., "Tonic") instead of h1 tag , because for the purpose of search and optimization it's recommend to have one h1 header tag for your website.

Reply:
Fixed.